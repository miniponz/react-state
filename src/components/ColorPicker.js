import React, { PureComponent } from 'react';

export default class ColorPicker extends PureComponent {
  state = {
    color: ''
  };

  render() {
    let color = this.state;
    
    return (
      <section>
        <h1>Pick Colors!</h1>
        <button>Purple</button>
        <button>Blue</button>
        <button>Green</button>
        <span></span>
      </section>

    );
  }
}
