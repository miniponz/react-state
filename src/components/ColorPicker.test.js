import React from 'react';
import { shallow } from 'enzyme';
import ColorPicker from './ColorPicker';

describe('color picker', () => {
  it('renders a color picker', () => {
    const wrapper = shallow(<ColorPicker />);
    expect(wrapper).toMatchSnapshot();
  });
});
