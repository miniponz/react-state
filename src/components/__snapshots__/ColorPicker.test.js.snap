// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color picker renders a color picker 1`] = `
<section>
  <h1>
    Pick Colors!
  </h1>
  <button>
    Purple
  </button>
  <button>
    Blue
  </button>
  <button>
    Green
  </button>
  <span />
</section>
`;
